### PR TITLE
configurable primary_medium for odin_checks and dynamic_table_commands check tables

### PR DESCRIPTION
**Pull request by kruftik**

### CODE REVIEW

File: yt/odin/checks/bin/dynamic_table_commands/__main__.py [hunk 1]
```diff
@@ -15,12 +15,13 @@ def run_check(yt_client, logger, options, states):
...
     temp_path = options["temp_tables_path"]
+    temp_table_primary_medium = options.get("temp_tables_primary_medium", "default")
 
     check_result = states.FULLY_AVAILABLE_STATE
 
     if not rpc_client.exists(temp_path):
-        rpc_client.mkdir(temp_path, recursive=True)
-        logger.info('Created temp directory %s', temp_path)
+        rpc_client.mkdir(temp_path, attributes={"primary_medium": temp_table_primary_medium}, recursive=True)
+        logger.info('Created temp directory %s (primary_medium=%s)', temp_path, temp_table_primary_medium)
```
Comment (Kontakter) [inline yt/odin/checks/bin/dynamic_table_commands/__main__.py:24]: `'.... (primary_medium: %s)'`

**Comment by kruftik** [comment]
@Kontakter , pls take another look

**Review by Kontakter — APPROVED** [review]
Looks good!

**Review by Kontakter — CHANGES_REQUESTED** [review]
This PR brokes some tests:

```
yt/odin/tests/test_db.py:83: in test_yt_db_for_cluster
    init_yt_table(TABLE, client, bundle="default")
E   TypeError: init_yt_table() missing 1 required positional argument: 'primary_medium'
```

**Comment by kruftik** [comment]
> This PR brokes some tests:
> 
> ```
> yt/odin/tests/test_db.py:83: in test_yt_db_for_cluster
>     init_yt_table(TABLE, client, bundle="default")
> E   TypeError: init_yt_table() missing 1 required positional argument: 'primary_medium'
> ```

just fixed it

**Comment by Kontakter** [comment]
Shipped!

**Review by Kontakter — CHANGES_REQUESTED** [review]
```
yt/odin/tests/test_db.py:72: in test_yt_db
    init_yt_table(TABLE, client, bundle="default")
E   TypeError: init_yt_table() missing 1 required positional argument: 'primary_medium'
```

Do you run all tests and check all usages?

**Comment by kruftik** [comment]
@Kontakter 

> ```
> yt/odin/tests/test_db.py:72: in test_yt_db
>     init_yt_table(TABLE, client, bundle="default")
> E   TypeError: init_yt_table() missing 1 required positional argument: 'primary_medium'
> ```
> 
> Do you run all tests and check all usages?

to be honest, not really :( but for now, I seem to have fixed all the mentions.

could you try again?

**Comment by Kontakter** [comment]
Shipped!

**Pull request closed without merging**